src/Middleware/Spa/SpaProxy/src/SpaDevelopmentServerOptions.cs

```diff
@@ -14,4 +14,6 @@ internal sealed class SpaDevelopmentServerOptions
     public TimeSpan MaxTimeout => TimeSpan.FromSeconds(MaxTimeoutInSeconds);
 
     public string WorkingDirectory { get; set; } = "";
+
+    public bool KeepRunning { get; set; }
```

Comment:
What is the option on the csproj file to enable this?

Reply:
It is SpaProxyKeepRunning. Sorry forgot to add it. Please check new commit